tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -28,67 +30,176 @@ func TestServiceEntry(t *testing.T) {
+			name:              "UNREACHABLE_www.bing.com_80_over_google_443",
+			config:            "testdata/networking/v1alpha3/service-entry-google.yaml",
+			url:               "http://www.bing.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_www.bing.com_80_over_bing_wildcard_80",
 			config:            "testdata/networking/v1alpha3/service-entry-wildcard-bing.yaml",
 			url:               "http://www.bing.com",
 			shouldBeReachable: true,
 		},
+		// note this will get a 301 move response when reachable
 		{
-			name:              "UNREACHABLE_bing.com_over_bing_wildcard_80",
+			name:              "UNREACHABLE_www_bing.com_443_over_bing_wildcard_80",
 			config:            "testdata/networking/v1alpha3/service-entry-wildcard-bing.yaml",
-			url:               "http://bing.com",
+			url:               "https://www.bing.com",
+			shouldBeReachable: false,
+		},
+		// test resolution NONE
+		{
+			name:              "REACHABLE_www.wikipedia.org_443_over_wikipedia_cidr_range_443",
+			config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
+			url:               "https://www.wikipedia.org",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_www.google.com_443_over_wikipedia_cidr_range_443",
+			config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
+			url:               "https://www.google.com",
 			shouldBeReachable: false,
 		},
-		// See issue https://github.com/istio/istio/issues/7869
-		//{
-		//	name:              "REACHABLE_wikipedia.org_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://www.wikipedia.org",
-		//	shouldBeReachable: true,
-		//},
-		//{
-		//	name:              "UNREACHABLE_google.com_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://google.com",
-		//	shouldBeReachable: false,
-		//},
-		{
-			name:              "REACHABLE_en.wikipedia.org_over_wikipedia_wildcard",
+		{
+			name:              "REACHABLE_en.wikipedia.org_443_over_wikipedia_wildcard_tls",
 			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
 			url:               "https://en.wikipedia.org/wiki/Main_Page",
 			shouldBeReachable: true,
 		},
 		{
-			name:              "REACHABLE_de.wikipedia.org_over_wikipedia_wildcard",
+			name:              "REACHABLE_de.wikipedia.org_443__over_wikipedia_wildcard_tls",
 			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
 			url:               "https://de.wikipedia.org/wiki/Wikipedia:Hauptseite",
 			shouldBeReachable: true,
 		},
 		{
-			name:              "UNREACHABLE_www.wikipedia.org_over_wikipedia_wildcard",
+			name:              "UNREACHABLE_www.wikipedia.org_443_over_wikipedia_wildcard_tls",
```

Comment:
Why is this unreachable?

Reply:
good q, I didn't touch this test other than making the name clear :).  I think because within the virtual services, it only refers to en and de with 100 weight each.

Reply:
So I believe this will be reachable if the two virtual services are removed in wildcard-tls-wikipedia.yaml, which is similar as the tests I added for wildcard-tls-google-no-vs.yaml

Reply:
I see, due to virtual services.

Reply:
yes!